src/Compatibility/ControlGallery/src/Issues.Shared/Issue15542.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.Maui.Controls.CustomAttributes;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Graphics;
+
+#if UITEST
+using Microsoft.Maui.Controls.Compatibility.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Microsoft.Maui.Controls.Compatibility.ControlGallery.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 15542, "[Bug] Shell.TitleView does not render on iOS 16",
+		PlatformAffected.iOS)]
+#if UITEST
+	[NUnit.Framework.Category(UITestCategories.Shell)]
+	[NUnit.Framework.Category(UITestCategories.TitleView)]
+#endif
+	public class Issue15542 : TestShell
```

Comment:
Can we port these tests to our `Controls.DeviceTests` ? 

I have a few examples of doing `TitleView` tests here

https://github.com/dotnet/maui/blob/6dc3d240b434b379b03cdd2c788e26e374c07705/src/Controls/tests/DeviceTests/Elements/Shell/ShellTests.cs#L433

Reply:
We have a device tests now there validating the same https://github.com/dotnet/maui/blob/main/src/Controls/tests/DeviceTests/Elements/Shell/ShellTests.iOS.cs#L205

From https://github.com/dotnet/maui/pull/12834